src/vcpkg/base/files.cpp

```diff
@@ -1594,7 +1621,7 @@ namespace vcpkg
                     if (stdfs::is_directory(b->path(), ec))
 #endif // VCPKG_USE_STD_FILESYSTEM
                     {
-                        ret.push_back(from_stdfs_path(b->path()));
```

Comment:
my concern is that these will be added to tarballs of binary caches; thoughts?

Reply:
On macOS we can use the `--exclude` options to exclude `.DS_Store` files from the archive

Reply:
So you add the arguments `.string_arg("--exclude").string_arg("*.DS_Store")` here:
https://github.com/microsoft/vcpkg-tool/blob/2059ef11aa6067e6f59b0d939c5d17e3c5c47d3e/src/vcpkg/binarycaching.cpp#L202-L207